zebra/rt_netlink.h

```diff
@@ -100,6 +102,10 @@ extern int netlink_macfdb_read_specific_mac(struct zebra_ns *zns,
 					    uint16_t vid);
 extern int netlink_neigh_read_specific_ip(const struct ipaddr *ip,
 					  struct interface *vlan_if);
+extern void netlink_handle_5549(uint32_t ndm_family, uint32_t ndm_state, struct zebra_if *zif,
+				struct interface *ifp, struct ipaddr *ip, bool handle_failed);
+
+extern bool is_mac_vni_mcast_group(struct ethaddr *mac, vni_t vni, struct in_addr grp_addr);
```

Comment:
same thing with this function, there is nothing netlink/kernel specific about this and should be in zebra_neigh.c